DEPRECATED.rst

```diff
@@ -568,6 +568,9 @@ deprecated in Release 1.64, and removed in Release 1.67. Use function
 list(SimpleFastaParser(handle)) from Bio.SeqIO.FastaIO instead (but
 ideally convert your code to using an iterator approach).
 
+Function 'GC' in Bio.SeqUtils was declared obsolete in Release 1.80.
```

Comment:
Not just obsolete, but deprecated https://biopython.org/wiki/Deprecation_policy

Reply:
Hm, my bad, will fix it.